stable/kong/ci/dbless-without-kic.yaml

```diff
@@ -1,5 +0,0 @@
-# CI test for testing dbless deployment
-env:
```

Comment:
Please leave this test case as is.
This test verifies the case of deploying Kong in DB-less mode without a config from a ConfigMap, to cater to use-cases where configuration is loaded via the Admin API `/config` endpoint.

Please do add the `-values` suffix to the filename.